lisa/tools/docker.py

```diff
@@ -0,0 +1,112 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from lisa.executable import Tool
+from lisa.operating_system import CentOs,Debian, Redhat, Ubuntu
+
+class Docker(Tool):
+
+    def _install_docker_engine(self):
```

Comment:
The tool support a _install method. Refer to document and other tools about how they implemented.